web/modules/custom/dcamp_attendees/templates/attendees-list.html.twig

```diff
@@ -25,6 +25,14 @@
           </div>
           <h2 class="attendee__title">{{ attendee.getName }}</h2>
           <h3 class="attendee__company">{{ attendee.getCompany }}</h3>
+          <div class="attendee__social">
+            {% if attendee.getTwitter %}
+             <a class="social-media__link social-media__link--twitter" href="{{ attendee.getTwitterUrl(attendee.getTwitter) }}" target="_blank"></a>
```

Comment:
@nuez I added these wrappers because sometimes we don't get a complete URL. `getTwitterUrl` and `getDrupalUrl` validate them.